pkg/plugins/runtime/k8s/webhooks/owner_reference_mutator_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
-	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	admissionv1 "k8s.io/api/admission/v1"
```

Comment:
WDYT about spinning out an issue to add `importas` rules to standardize common Kubernetes import aliases? Thinking `core/v1`, `meta/v1` and friends ...

Reply:
https://github.com/kumahq/kuma/issues/2811